src/lib/accessibility/Behaviors/Menu/MenuBehavior.ts

```diff
@@ -1,11 +1,19 @@
-import { Accessibility } from '../../interfaces'
+import { Accessibility, FocusZoneMode } from '../../interfaces'
 
 const MenuBehavior: Accessibility = {
   attributes: {
     root: {
       role: 'menu',
     },
   },
+  focusZone: {
+    mode: FocusZoneMode.Wrap,
+    props: {
+      onActiveElementChanged: (element, ev) => {
+        console.error('MENU BEHAVIOR', 'on active element changed', 'element', element, 'ev', ev)
```

Comment:
Do we need to have this? We'll get many errors into console then :)

Reply:
I think for now you can set in props ``` isCircularNavigation: true```, as it's a requirement by ARIA documentation
It might be in separate PR, but would be good to have VerticalMenuBehavior as we have for VerticalMenuItemBehavior, so then in props can be set directions:
``` 
 props: {
      direction: FocusZoneDirection.horizontal, // vertical for Vertical behavior
      isCircularNavigation: true,     // 
    },
```

Reply:
and horizontal/vertical menu should be in a separate PR. for now let's just include isCircularNavigation